### ISSUE TITLE
Accessibility Audit

### ISSUE THREAD

**Issue by sydney-devine** [Gary-Community-Ventures/benefits-calculator#1025]

**Comment by ivonne-hernandez** [comment]
- Step 2 
	- [ ] [Ensures the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds](https://axe.deque.com/issues/f3980ebb-4d45-4edb-a4e9-5d7c86af8e31) . The Public Charge link and it's background need a higher contrast ratio
		![[Screenshot 2024-04-22 at 1.22.02 PM.png]]
	- [ ] [Buttons must have discernible text](https://axe.deque.com/issues/62ee7d92-e5e5-4845-b6c0-fd953e470646). ![[Screenshot 2024-04-22 at 1.21.44 PM.png]]![[Screenshot 2024-04-22 at 1.24.31 PM.png]]
	![[Screenshot 2024-04-22 at 1.27.04 PM.png]]![[Screenshot 2024-04-22 at 1.41.55 PM.png]]

Results page (19+ issues):
1. Seven of the issues entail the Category icons having an aria-label attribute within a div that has no valid role attribute ![[Screenshot 2024-04-22 at 1.46.45 PM.png]]
direct link to this issue: https://axe.deque.com/issues/2bdc43c9-b368-4544-bf78-8f7463ad59fc
2. Document should have one main landmark [link to how to fix it](https://dequeuniversity.com/rules/axe/4.8/landmark-one-main?application=AxeChrome)
3. Page should contain a level-one heading [link to how to fix it](https://dequeuniversity.com/rules/axe/4.8/page-has-heading-one?application=AxeChrome)
4. 10+ issues of this [All page content should be contained by landmarks](https://axe.deque.com/issues/65a41ed6-0acc-4e26-b323-1dd2316cfe44)![[Screenshot 2024-04-22 at 1.59.57 PM.png]]![[Screenshot 2024-04-22 at 2.00.34 PM.png]]![[Screenshot 2024-04-22 at 2.00.48 PM.png]]![[Screenshot 2024-04-22 at 2.02.37 PM.png]]![[Screenshot 2024-04-22 at 2.02.55 PM.png]]![[Screenshot 2024-04-22 at 2.03.24 PM.png]]
![[Screenshot 2024-04-23 at 10.00.45 AM.png]]
![[Screenshot 2024-04-23 at 10.03.19 AM.png]]
![[Screenshot 2024-04-23 at 10.31.13 AM 2.png]]
- Program page has 5 issues, 1 Document should have one main landmark, 4 All page content should be contained by landmarks
	![[Screenshot 2024-04-22 at 2.06.00 PM.png]]
- For Near Term Benefits, 1 Document should have one main landmark, 1 Page should contain a level-one heading, 19 All page content should be contained by landmarks ![[Screenshot 2024-04-22 at 2.09.59 PM.png]]

**Comment by ivonne-hernandez** [comment]
Link to Google doc:
https://docs.google.com/document/d/1Q9yk3SOkrT-cSLYvhjbuLyX64SLDEU10gueR4_DnNsY/edit
